static/src/stylesheets/module/nav/_navigation.scss

```diff
@@ -678,7 +679,7 @@ $c-navigation-expandable-background: colour(neutral-1);
 
 // TODO: Move
 .navigation-toggle {
-    z-index: 2;
+    z-index: 103;
```

Comment:
I don't think it should be _that_ high. I set it there to be sure it'd have the desired effect, but will see if I can drop it to something lower.